src/breadcrumb-group/full-dropdown.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { Ref } from 'react';
+import clsx from 'clsx';
+
+import { CustomTriggerProps } from '../button-dropdown/interfaces';
+import InternalButtonDropdown from '../button-dropdown/internal';
+import InternalIcon from '../icon/internal';
+import { DATA_ATTR_FUNNEL_KEY, DATA_ATTR_RESOURCE_TYPE, FUNNEL_KEY_FUNNEL_NAME } from '../internal/analytics/selectors';
+import { CancelableEventHandler } from '../internal/events';
+import { BreadcrumbGroupProps } from './interfaces';
+
+import styles from './styles.css.js';
+
+interface FullCollapsedDropdownProps {
+  items: ReadonlyArray<BreadcrumbGroupProps.Item>;
+  onClick: CancelableEventHandler<{ id: string }>;
+  onFollow: CancelableEventHandler<{ id: string }>;
+}
+
+const metadataTypeAttribute = {
+  [DATA_ATTR_RESOURCE_TYPE]: 'true',
+};
+
+export const FullDropdown = ({ items, onClick, onFollow }: FullCollapsedDropdownProps) => (
+  <>
+    <InternalButtonDropdown
+      items={items.map((item, index) => {
+        const isCurrentPage = index === items.length - 1;
+        return {
+          id: index.toString(),
+          text: item.text,
+          href: isCurrentPage ? undefined : item.href,
+          disabled: isCurrentPage,
+          isCurrentPage,
+        };
+      })}
+      className={styles['collapsed-button-dropdown']}
+      customTriggerBuilder={getFullDropdownTrigger(items[items.length - 1].text)}
+      linkStyle={true}
+      onItemClick={onClick}
+      onItemFollow={onFollow}
+      analyticsMetadataTransformer={metadata => {
+        if (metadata.detail?.id) {
+          delete metadata.detail.id;
+        }
+        if (metadata.detail?.position) {
+          metadata.detail.position = `${parseInt(metadata.detail.position as string, 10) + 1}`;
+        }
+        return metadata;
+      }}
+    />
+    {items.length >= 1 && (
+      <span className={styles.hidden} {...metadataTypeAttribute}>
+        {items[1].text}
```

Comment:
Should this be `items[items.length - 1].text`?

Reply:
No, it's supposed to specifically be the second item, it corresponds to this line https://github.com/cloudscape-design/components/blob/49a178d8f7fa35206645d217c949ecb180dc322d/src/breadcrumb-group/item/funnel.tsx#L29

Will add a comment to clarify